### PR TITLE
doc: Updated pinmapping diagram

### PR DESCRIPTION
**Pull request by HU90m**
1) The previous mapping had the pmod sda and scl pins the wrong way around.
2) The `selsw` has been added, previously forgotten.
3) mikroBUS doesn't specify GPIO functionality, so the gpio_mb has been removed.

### CODE REVIEW

File: doc/ip/pinmux/pin_mappings.dot [hunk 1]
```diff
@@ -438,7 +445,8 @@ digraph {
   pwm:2 -> pmod0:6;
   pwm:3 -> pmod1:2;
   pwm:4 -> pmod1:6;
-  pwm:5 -> lcd:backlight;
```
Comment (marnovandermaas) [inline doc/ip/pinmux/pin_mappings.dot:441]: what's connected to pwm 5?

**Review by marnovandermaas — APPROVED** [review]
Looks reasonable to me just one question.

File: doc/ip/pinmux/pin_mappings.dot (hunk 1 above)
Comment (HU90m) [reply doc/ip/pinmux/pin_mappings.dot:441]: `rph:g21_spi1_sclk` and `ah:tmpio11`

Comment (marnovandermaas) [reply doc/ip/pinmux/pin_mappings.dot:441]: Great thanks!

**Pull request merged**